.gitignore

```diff
@@ -1,77 +1,77 @@
-# Created by .ignore support plugin (hsz.mobi)
-### Python template
-# Byte-compiled / optimized / DLL files
-__pycache__/
-*.py[cod]
-*$py.class
-
-# C extensions
-*.so
-
-# Distribution / packaging
-.Python
-build/
-develop-eggs/
-dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-wheels/
-
-# Installer logs
-pip-log.txt
-pip-delete-this-directory.txt
-
-# Unit test / coverage reports
-htmlcov/
-.tox/
-.coverage
-.coverage.*
-.cache
-nosetests.xml
-coverage.xml
-*.cover
-.hypothesis/
-
-# Django stuff:
-*.log
-.static_storage/
-.media/
-settings_local.py
-
-# Sphinx documentation
-docs/_build/
-
-# PyBuilder
-target/
-
-# Jupyter Notebook
-.ipynb_checkpoints
-
-# pyenv
-.python-version
-
-# celery beat schedule file
-celerybeat-schedule
-
-# SageMath parsed files
-*.sage.py
-
-# Environments
-.env
-.venv
-env/
-venv/
-ENV/
-env.bak/
-venv.bak/
-
-*.sqlite3
-
-# IDE
-.idea
+# Created by .ignore support plugin (hsz.mobi)

```

Comment:
@PaulakaPaul, please do not forget to undo this permission changes.

Reply:
Ok, I will undo it.

Reply:
I don't want so sound stupid, but I will have to. What permission changes ? I looked over my commit, but I don't get it.

Reply:
@PaulakaPaul, isn't it weird that your PR looks like you made some changes (https://github.com/safefleet/labs-server/pull/10/files) in files like `.gitignore` but actually you didn't make any change ? You probably changed some file permissions.